doc/contributing.md

```diff
@@ -16,37 +16,208 @@ and we encourage all to read it carefully.
+After you've submitted a pull request, you should expect to hear at least a
+comment within a couple of days. We may suggest some changes, improvements or
+alternative implementation details.
+
+To increase the chances of getting your pull request accepted quickly, try to:
+
+* Submit a friendly PR
+  - Write a good and detailed description of what the PR does.
+  - Write some documentation for your code (docstrings) and leave comments
+    explaining the *reason* behind non-obvious things.
+  - Write tests for the code you wrote/modified if needed.
+    Please refer to [Testing your code](#testing-your-code) or
+    [Testing plots](#testing-plots).
+  - Include an example of new features in the gallery or tutorials.
+    Please refer to [Gallery plots](#gallery-plots) or [Tutorials](#tutorials).
+* Have a good coding style
+  - Use readable code, as it is better than clever code (even with comments).
+  - Follow the [PEP8](http://pep8.org) style guide for code and the
+    [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html)
+    for docstrings. Please refer to [Code style](#code-style).
+
+Pull requests will automatically have tests run by GitHub Actions.
+This includes running both the unit tests as well as code linters.
+GitHub will show the status of these checks on the pull request.
+Try to get them all passing (green).
+If you have any trouble, leave a comment in the PR or
+[get in touch](#how-can-i-talk-to-you).
+
+## Contributing Documentation
+
+### PyGMT Documentation Overview
 
-## Reporting a Bug
+There are four main components to PyGMT's documentation:
 
-Find the *Issues* tab on the top of the GitHub repository and click *New Issue*.
-You'll be prompted to choose between different types of issue, like bug reports and
-feature requests.
-Choose the one that best matches your need.
-The Issue will be populated with one of our templates.
-**Please try to fillout the template with as much detail as you can**.
-Remember: the more information we have, the easier it will be for us to solve your
-problem.
+* Gallery examples, with source code in Python `*.py` files under the
+  `examples/gallery/` folder.
+* Tutorial examples, with source code in Python `*.py` files under the
+  `examples/tutorials/` folder.
+* API documentation, with source code in the docstrings in Python `*.py`
+  files under the `pygmt/src/` and `/pygmt/datasets/` folders.
+* Getting started/developer documentation, with source text in ReST `*.rst`
+  and markdown `*.md` files under the `doc/` folder.
 
+The documentation are written primarily in
+[reStructuredText](https://docutils.sourceforge.io/rst.html) and built by
+[Sphinx](http://www.sphinx-doc.org/). Please refer to
+[reStructuredText Cheatsheet](https://docs.generic-mapping-tools.org/latest/rst-cheatsheet.html)
+if you are new to reStructuredText. When contributing documentation, be sure to
+follow the general guidelines in the [pull request workflow](#pull-request-workflow)
+section.
 
```

Comment:
How about adding a new paragraph to indicate that we have two ways to contribute to the documentation?
- Editing the Documentation on GitHub
- Editing the Documentation locally

Reply:
Good idea, I updated for this and your comment below in https://github.com/GenericMappingTools/pygmt/pull/1339/commits/b88f185f4d371d162764bb92b61e505af0ecfefd.